### DISCUSSION TITLE
Missing Integrations

### DISCUSSION THREAD

**Discussion by Bae360** [hcengineering/platform#5712] [Q&A]
I have just installed the [huly-selfhost](https://github.com/hcengineering/huly-selfhost) 
When I go /setting/integrations, I see only two types of integrations
![image](https://github.com/hcengineering/platform/assets/135948595/b7336a3f-340a-43df-92d9-e725d1009da9)
How can we add the others ?

**Comment by aonnikov** [comment]
I believe that's all that we support for selfhosted version.
